src/mmw/js/src/compare/controllers.js

```diff
@@ -16,28 +18,87 @@ var CompareController = {
             App.currProject
                 .fetch()
                 .done(function() {
+                    setupProjectCopy();
```

Comment:
Got it. Thanks.

> the original project (without changes) is saved when logging in

Consider asking the user to log in before going to compare view. I may have missed some discussion around this, but I don't see much value in not saving the project by this point.

> changes to the results and inputs are not present after hitting the back button (and project is unsaved)

Consider fetching the project from the server when returning from the compare view to repopulate the page with the saved version of the project.

Alternatively, consider doing the project "copying" a bit more dynamically and on an as-needed basis. For example, once a precipitation slider is changed for a scenario, have that scenario store it's current attributes in a temporary attribute. When the page is moved back to model, then apply the saved attributes to the changed scenario. I haven't used it much but `model.previousAttributes` and/or `model.previous()` may be of some use here.